src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -91,6 +91,11 @@ AsyncRPCOperation_sendmany::AsyncRPCOperation_sendmany(
             throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, string("runtime error: ") + e.what());
         }
     }
+
+    // Log the context info i.e. the call parameters to z_sendmany
+    if (LogAcceptCategory("zrpcunsafe")) {
+        LogPrint("zrpcunsafe", "%s: starting (%s)\n", getId(), json_spirit::write_string( contextInfo, false));
```

Comment:
Done, but now labelled as 'initialized' since the z_sendmany op is queued and might not start processing for a while.